### PR TITLE
Harmonize parameter names

### LINKED ISSUE #166: Harmonize all parameter names in tests and functions

### PR DESCRIPTION
**Pull request by xtrojak**
This PR is trying to harmonize parameter names across the package. Particular goal was to al least ensure that all backtrace function calls from `unsupervised` and `hybrid` use the same name for the same parameter.

Close #166.

### CODE REVIEW

File: R/adaptive.bin.R [hunk 1]
```diff
@@ -15,19 +15,19 @@ compute_densities <- function(masses, tol, weighted, intensities, bw_func, n = 5
...
 
   all.mass.turns <- find.turn.point(all.mass.den$y)
   all.mass.vlys <- all.mass.den$x[all.mass.turns$vlys]
   return(all.mass.vlys)
 }
 
 #' @export
-compute_breaks <- function(tol, masses, intensi, weighted) {
-  all.mass.vlys <- compute_mass_values(tol, masses, intensi, weighted)
+compute_breaks <- function(mz_tol, masses, intensi, intensity_weighted) {
```
Comment (maximskorik) [inline R/adaptive.bin.R]: Any idea what that `intensi` parameter might be? The naming is rather obscure, especially next to the `intensity_weighted` parameter.

Comment (maximskorik) [reply R/adaptive.bin.R]: It's actually documented here: https://github.com/RECETOX/recetox-aplcms/pull/167/files#diff-8cbac6537922b087d1357bd498791f7ad437b8e5cae09a9f8189dd21c496cef9R66

Maybe it's a good idea to rename it.

**Review by maximskorik — APPROVED** [review]
Great job!

File: R/adaptive.bin.R (hunk 1 above)
Comment (xtrojak) [reply R/adaptive.bin.R]: Good idea, maybe something like `intensity_binned`?

Comment (xtrojak) [reply R/adaptive.bin.R]: Done in [a6dadc5](https://github.com/RECETOX/recetox-aplcms/pull/167/commits/a6dadc592527e80302c1ddc322c9921739edce22).

**Review by hechth — APPROVED** [review]
Great work - thanks so much for taking this on!

**Pull request merged**